app/main/posts/views/filters/filters-dropdown.html

```diff
@@ -17,7 +17,11 @@
         <post-active-search-filters ng-model="filtersVar" filters-var="filtersVar"></post-active-search-filters>
     </div>
     <div class="form-field">
-        <button type="button" class="button button-alpha button-right" ng-click="saveSavedSearchModal()">Save search</button>
+        <button type="button" class="button button-alpha button-right" ng-click="saveSavedSearchModal()" translate="set.save_savedsearch">Save search</button>
+    </div>
+
+    <div class="form-field">
+        <button type="button" class="button button-alpha button-right" ng-click="editSavedSearchModal()" ng-show="canUpdateSavedSearch" translate="set.update_savedsearch">Edit Saved search</button>
```

Comment:
this is shown only when you can update the saved search, meaning you have selected one and your permissions are OK